dev/com.ibm.websphere.appserver.features/visibility/public/appSecurityClient-2.0/resources/l10n/io.openliberty.appSecurityClient-2.0.properties

```diff
@@ -0,0 +1,18 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=Support for securing the client container runtime environment \
+ and applications. This feature enables application security, CSIv2 and SSL on the client.
```

Comment:
needs a serial comma

This feature enables application security, CSIv2, and SSL on the client.

Reply:
Updated the text.